app.js

```diff
@@ -155,31 +163,33 @@ function LeftButton(route, navigator, index, navState) {
       if (index === 1) {
         backTitle = 'Home'
       }
-      return (
-        <TouchableOpacity
-          style={styles.backButton}
-          onPress={() => navigator.pop()}
-        >
-          <Icon style={styles.backButtonIcon} name='ios-arrow-back' />
-          <Text style={styles.backButtonText}>{backTitle}</Text>
-        </TouchableOpacity>
-      )
+      if (Platform.OS === 'android') {
```

Comment:
Could you change the `if` to only operate on the icon, instead of swapping out the entire touchableOpacity?

Like, `name={Platform.OS === 'android' ? 'md-arrow-back' : 'ios-arrow-back'}`. 

Thanks.